PopulateTools/gobierto
# 2904 endopoint for data retrieving

Closes #2904


## :v: What does this PR do?

- Adapt Perspective to use CSV instead of JSON
- Default query SELECT * FROM table LIMIT 50
- Calculate response time with Javascript
- Remove the number of rows from the editor.
- Show private public icon on every visualization.
- Include button to remove every Visualization.
- Avoid multiple requests in datasets.
- When we switch between datasets the query must remain in each of the editor
- Disabled keyboard shortcuts while try to saving a query or visualization.

## :mag: How should this be manually tested?

Staging
DevTools open with network panel.

Steps:

Multiple requests:
1. Go to any dataset, run a query, you should see only a request to API.
2. Goes to another dataset, run a query and check again network panel, you should see a unique request.

Disable shortcuts:
1. Try to saving a query with letters c, r(shortcuts to display modal queries), shortcuts dont be fired and the request should be unique.

Visualizations tab:
1. Go to editor and save two or more visualizations(at least one of them private)
2. Go to Visualization tab, you should see icons for public and private visualizations, also another button to delete any visualization.
